src/utils/utils.ts

```diff
@@ -4,8 +4,45 @@
 import { Stack } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 
-export const setTemplateDesc = (construct: Construct, sbtDesc: string) => {
+export const addTemplateTag = (construct: Construct, tag: string) => {
   const stackDesc = Stack.of(construct).templateOptions.description;
-  let description = stackDesc === undefined ? sbtDesc : stackDesc + ' - ' + sbtDesc;
+  const baseTelemetry = 'SaaS Builder Toolkit (uksb-1tupboc57)';
```

Comment:
should we use the full name here? or would the smaller one be better to conserve space? (`sbt-aws`)